### ISSUE TITLE
Open up root

### ISSUE THREAD

**Issue by adamierymenko** [zerotier/edge#1]
Since this is becoming an OSS product/project, open 'sudo' in the distro on next update.

**Comment by adamierymenko** [comment]
Also a reboot button/switch on UI?

**Comment by cferrey** [comment]
@adamierymenko Yes, please!  My ZTe has been locking up remotely/losing its ZT connection.  It drops off off the WAN/LAN/ZT network semi-frequently, and I have no way to remotely reset it.

**Comment by cron410** [comment]
I would also like sudo access on this device. 

Can anyone tell me what version the Edge is currently at? I just bought someone's device that was never put online, and will not pull updates. It's running 0.5.0.

**Comment by genieinfo** [comment]
Same here, i was never able to update my 2 edges. 
Still stuck @ 0.5.0.
It has never been able to update by itself, saying 0.5.0 is the most recent version even though it seems it was released in nov. 2018
Would you please allow sudo and/or permit installing other distro on these quite capable Marvell-based devices ?
Many thanks in advance if you do so

**Comment by cron410** [comment]
It's never going to update because the updates only come from this Git repo. I was able to gain `root` access by taking the SD card out and mounting it on another linux system, removed the password hashes in `shadow` and `passwd` files, and temporarily enabled passwordless logins in the `sshd_config` file along with `root` login. If you are able to do those basic things, you should absolutely do it. I've been attempting to update my Edge device through the Alpine APK system without success. Next I'm looking at applying this Git repo to a fresh install of Alpine on the espressobin board.

**Comment by genieinfo** [comment]
Many thanks for sharing your attempts unlocking these espresso bin boards.I must admit I gave up and ended setting up some raspberry pi 4 for my testing and even production at some facilities.That said, I absolutely didn't regret crowdfunding Zerotier and even subscribed to their offer.Now I mix Wireguard and Zerotier according to my needs.SebastienLe 24 sept. 2021 01:36, cron410 ***@***.***> a écrit :
It's never going to update because the updates only come from this Git repo. I was able to gain root access by taking the SD card out and mounting it on another linux system, removed the password hashes in shadow and passwd files, and temporarily enabled passwordless logins in the sshd_config file along with root login. If you are able to do those basic things, you should absolutely do it. I've been attempting to update my Edge device through the Alpine APK system without success. Next I'm looking at applying this Git repo to a fresh install of Alpine on the espressobin board.

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

Le 24 sept. 2021 01:36, cron410 ***@***.***> a écrit :
It's never going to update because the updates only come from this Git repo. I was able to gain root access by taking the SD card out and mounting it on another linux system, removed the password hashes in shadow and passwd files, and temporarily enabled passwordless logins in the sshd_config file along with root login. If you are able to do those basic things, you should absolutely do it. I've been attempting to update my Edge device through the Alpine APK system without success. Next I'm looking at applying this Git repo to a fresh install of Alpine on the espressobin board.

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

Le 24 sept. 2021 01:36, cron410 ***@***.***> a écrit :
It's never going to update because the updates only come from this Git repo. I was able to gain root access by taking the SD card out and mounting it on another linux system, removed the password hashes in shadow and passwd files, and temporarily enabled passwordless logins in the sshd_config file along with root login. If you are able to do those basic things, you should absolutely do it. I've been attempting to update my Edge device through the Alpine APK system without success. Next I'm looking at applying this Git repo to a fresh install of Alpine on the espressobin board.

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

Le 24 sept. 2021 01:36, cron410 ***@***.***> a écrit :
It's never going to update because the updates only come from this Git repo. I was able to gain root access by taking the SD card out and mounting it on another linux system, removed the password hashes in shadow and passwd files, and temporarily enabled passwordless logins in the sshd_config file along with root login. If you are able to do those basic things, you should absolutely do it. I've been attempting to update my Edge device through the Alpine APK system without success. Next I'm looking at applying this Git repo to a fresh install of Alpine on the espressobin board.

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

ngLe 24 sept. 2021 01:36, cron410 ***@***.***> a écrit :
It's never going to update because the updates only come from this Git repo. I was able to gain root access by taking the SD card out and mounting it on another linux system, removed the password hashes in shadow and passwd files, and temporarily enabled passwordless logins in the sshd_config file along with root login. If you are able to do those basic things, you should absolutely do it. I've been attempting to update my Edge device through the Alpine APK system without success. Next I'm looking at applying this Git repo to a fresh install of Alpine on the espressobin board.
